src/index.scss

```diff
@@ -0,0 +1,5 @@
+body {
+    div#root{
+      background-color: rgb(255, 255, 255);
+    }
+}
```

Comment:
@e-ian great work 
please add a new line at the end of this file though

Reply:
Done @patrickf949